plugins/auth/public/auth-oauth2/README.md

```diff
@@ -63,3 +63,13 @@ const CopyWebpackPlugin = require('copy-webpack-plugin');
   ]
 }
 ```
+
+For more simple scenarios such as Luigi setup without a framework, you can use any copy plugin to copy the `callback.html` file. An example is given below using [copyfiles](https://www.npmjs.com/package/copyfiles):
```

Comment:
```suggestion
In case of a basic Luigi setup without a framework, you can use any copy plugin to copy the `callback.html` file and then modify the `package.json` script to copy the file when building. One package that could be helpful is [copyfiles](https://www.npmjs.com/package/copyfiles). Following is an example:
```